src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -50,6 +51,7 @@ static void SendCommandToWorkersOutsideTransaction(TargetWorkerSet targetWorkerS
 												   const char *command, const char *user,
 												   bool
 												   failOnError);
+bool IsWorkerTheCurrentNode(WorkerNode *workerNode);
```

Comment:
we already externally declared `IsWorkerTheCurrentNode`, so probably this is redundant (and slightly confusing)